harmonizer/Cargo.toml

```diff
@@ -1,26 +1,26 @@
 [package]
 name = "harmonizer"
 version = "0.33.0"
-authors = ["Apollo Graph, Inc. <opensource@apollographql.com>"]
+authors = [ "Apollo Graph, Inc. <opensource@apollographql.com>" ]
 edition = "2018"
 description = "Apollo Federation utility to compose a supergraph from subgraphs"
 homepage = "https://www.apollographql.com/apollo-federation/"
 documentation = "https://apollographql.com/docs/federation/"
 repository = "https://github.com/apollographql/federation/"
 license = "MIT"
 readme = "README.md"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-include = ["dist/**/*", "js/**/*", "src/**/*", "Cargo.toml"]
+include = [ "dist/**/*", "js/**/*", "src/**/*", "Cargo.toml" ]
 
 [dependencies]
 anyhow = "1.0.39"
 deno_core = "0.86.0"
-serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.60"
 thiserror = "1.0.23"
 
+  [dependencies.serde]
+  version = "1.0"
+  features = [ "derive" ]
+
```

Comment:
🤔Looks like the JS TOML lib eats comments for breakfast (not surprising) and prefers the dot notation for nested hash maps.  Luckily our only comments in this file right now is totally unnecessary boilerplate￼. 

Let's definitly try to this but I feel like this could end up being a tooling battle.

Reply:
Yeah, I included the changes we can expect from toml parser to make sure this didn't go unnoticed. I agree it's a bit of a bummer.

On that note, the library I've selected is current-ish but hasn't seen much love in the last 6 months. Could be fun to write a TOML parser/printer 😄